nimbus-ui/nimbusui/src/app/components/platform/form/elements/action-dropdown.component.mockdata.ts

```diff
@@ -0,0 +1,60 @@
+export const ActionDropdownLink = {
```

Comment:
can you name the file *.spec.ts

Reply:
Just confirming, you mean to rename it: `action-dropdown.component.mockdata.spec.ts`?

Reply:
yes, that way we know that it is test file

Reply:
Renamed!